src/styles.css

```
@@ -709,3 +709,83 @@ footer a:hover{text-decoration:underline}
+/* Desktop / Default Layout */
+.nav-inner {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  gap: 20px;
+}
+
+/* Navigation Links */
+.nav-links,
+#mobileMenu {
```

Stylelint (17.12.0)

[error] Expected id selector "`#mobileMenu`" to be kebab-case (selector-id-pattern)

(selector-id-pattern)